user_manual/files/webgui/sharing.rst

```diff
@@ -40,6 +40,22 @@ with the user "pierpont", who can *share*, *edit*, *create*, *change*, and
 .. figure:: ../../images/files_page-2.png
    :alt: Sharing files.
 
+What Happens When Share Recipients Move Files and Folders?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If a share recipient has "**can edit**" privileges and moves files or folders out of the share, ownCloud restores a backup copy of the moved file/folder in the Deleted Files (Trash) of the share’s owner. 
+The user who moved the file/folder out of the share still has the original copy there, along with its attached metadata.
+
+That way, the files/folders are not permanently lost.
+These files and folders can be restored to their original location, by clicking the **Restore** link, next to the respective file or folder.     
```

Comment:
restored by the share owner ? here it's not clear about who is doing the restoring since we're talking from an outside perspective

Reply:
On reading it through a few times, you're right; will fix.